README.md

```diff
@@ -1,24 +1,125 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
 
-Things you may want to cover:
+  <img src="https://user-images.githubusercontent.com/93520178/203661672-538bff13-8797-4d20-9b82-372aa011d464.png" alt="logo" width="100"  height="auto" />
+  <br/>
```

Comment:
### **Optional**

- [ ] Please delete the microverse logo from your REAME file.